src/App.tsx

```diff
@@ -1,24 +1,233 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+/* eslint-disable no-param-reassign */
+import React, {
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {
+  createTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+} from './api/todos';
+import { AuthContext } from './components/Auth/AuthContext';
+import { NewTodoField } from './components/NewTodoField/NewTodoField';
+import { TodoFilters } from './components/TodoFilters/TodoFilters';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import {
+  ErrorNotification,
+} from './components/ErrorNotification/ErrorNotification';
+import { ErrorType } from './types/ErrorType';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [isUploadError, setIsUploadError] = useState(false);
+  const [error, setError] = useState(ErrorType.None);
+  const [isError, setIsError] = useState(false);
```

Comment:
you don't need these 2 values here. you can just simplify the logic and use `error` only